### DISCUSSION TITLE
how to keep highlighted option of OptionList, if the focus got to another widget

### DISCUSSION THREAD

**Discussion by MohamedElgamal** [Textualize/textual#3353] [Q&A] [answered]
I’m encountering a minor issue with the OptionList widget. When I select an item, it gets highlighted as expected. However, if the mouse focus shifts to another widget, the highlighted option reverts to its normal color. I would like the selected option to remain highlighted even when the focus is on another widget. Any assistance would be greatly appreciated

**Comment by TomJGooding** [comment, marked as answer]
See #3343
